### DISCUSSION TITLE
HELP SOS!!

### DISCUSSION THREAD

**Discussion by liagraham** [community/community#51345] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I can not deploy any of your apps

The description is very vague
what do I do???????

**Comment by creyD** [comment]
Does it have something to do with: https://www.githubstatus.com/?

**Comment by NJacksonDev** [comment]
I would take a look at the GitHub docs (see below) - they may provide you with the necessary information. 

https://docs.github.com/en
